app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False):
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return "{Name: " + self.name + \
+               ", Health: " + str(self.health) + \
+               ", Hidden: " + str(self.hidden) + "}"
+
+
+class Herbivore(Animal):
+
+    def hide(self):
+        if not self.hidden:
+            self.hidden = True
+        else:
```

Comment:
you can write this without `else`

Reply:
I'm not sure how. If you mean this construction, then it will not work. Because the new True value will be overwritten by False
![изображение](https://user-images.githubusercontent.com/70214623/176420611-79fe7980-ee25-4a92-a908-ec26cceb6477.png)

Reply:
Cunningly ))